src/main/java/uk/gov/hmcts/cmc/claimstore/services/UserService.java

```diff
@@ -10,9 +14,13 @@
 public class UserService {
 
     private IdamApi idamApi;
+    private final AuthTokenGenerator cachedServiceAuthTokenGenerator;
```

Comment:
Rename, it does not matter whether this is cached or not in the service context.

Reply:
renamed